asserts/trusted.go

```diff
@@ -0,0 +1,112 @@
+
+openpgp wsNcBAABCgAGBQJXaaGQAAoJENSlW+qX2DcgBHsgAMGkOBUBN51XkY85dRCKFuwoURE4
+lpJQAyFKMinEoG4MOsOCUX5oQgMDNXHenD6+xNWFit/YkI3flCrCP7yuPo0Vv95J8CI13HNpIiKo
+BbFH1DvZ7Lawi8ZzMHmbRZ7FImYjeMVbLMCN9ELJoXfuRR5B+80RRWADSGys5rPzum+J2R61ZH0k
+dSn5cFClEawELVCWDZQ1KiWjzNZl6utU02/l6Fu12LpYQMexh2t8n7jXFEmC8XLgxOLBVPOpfDXj
+WUhEKzxaTHjEjvCWIKyXY5lVsy6CVYV0UcZqQypz7o7juKyF7Pi1zNP/hgSMwPvzuKQNPV61zs/f
+NGwhomSfHUFFwJ7BfSqz5XaP5BkjM7SITrlu10ak+D2io+HWuKT+EYKl18ozjPuixgg7YbH7mk4t
+gst8faLorAiTaH0yJLF1xyZF/f280JyOL05W/47QLHZl9Bwc+PNR7c+IaVxkRZZbld8pp04D2iGY
+nhz3685hFXTZlhRSs6ROwmNuR2n/SelhQSyiUdyg/1qvNtC8hpetezBGNO9Q5+bQYxUI25NZ8V3X
+eDkijqyP+ygfbjkDWTNpW+CjM81I/8aYw+780iEsYauANfBqWVBCXVn0t84YQXWYJIDnpT3dLDsM
+9sUufA2cOV7nwEmgI9TNF0t/c2ermNd1tiwZUPy1QiGZqDO81RTNSEHgYDioppQihRdOa4PRCvfn
+7MefoJkReL0k2WYb1LVgJqZPBt9jatPlBTxbyQene6KwZ7S3mzv0moWkDkzcY+Jxopfd++ICcWTv
+lXaC4jf6f/l3Z64x+zpuWuR5j8Z7Z/+UTtDXAIpugjkx/1bV9gh8EnIOMGY5dvBl/Fzpw+d0xAsq
+P8OgG5IyBMcwcY2mqwcBsYxSOZARp9neWwN/76RidgIbDCaKtUx6gV83ryc0+vdPutAC65qmg1Dd
+/O53z6R3x7vNkHU3b3sd7cYKAmS8zQXTUUC3iaxnxUlEoaMcuZOGLvUKrjTHgD8f7WGqgvu65Du/
+GmbsUhgbhzAcRG8mAypCoLEUTHMF3mBP57A4wcoSw3UEuAMlk1jKqpjkRDhLIvueKU7tOqOu34Xl
+Ue9lxd2+dK06NkJMYh0IslvjsmcNZzgcO/irgOFEFI9FWDC6FXgK2Y8VB5fBe0ZtIUz+gAphjGOC
+ioqTwXYI4abZIWzzAlB+3yCyU9dwl4wc5zkD60DaTJDXuYpN/yA8Dw3FqUR2gjHwKcI4cI4kEamp
+MoGVNcH7Qq2DRFrFkdu/ebB9q0i6oOnZ6LXkVOmMgr+4lSOl8AeqbjZqxJOwppCNemVRXASnXzmk
+FnH2O7WswtPHB16rpFBt6fVbrrWVkQzVTbBtDcamn1GGSX8=
+`
+)
+
+var trustedAssertions []Assertion
+
+func init() {
+	canonicalAccount, err := Decode([]byte(encodedCanonicalAccount))
+	if err != nil {
+		panic(fmt.Sprintf("cannot decode trusted assertion: %v", err))
+	}
+	canonicalRootAccountKey, err := Decode([]byte(encodedCanonicalRootAccountKey))
+	if err != nil {
+		panic(fmt.Sprintf("cannot decode trusted assertion: %v", err))
+	}
+	trustedAssertions = []Assertion{canonicalAccount, canonicalRootAccountKey}
+}
+
+// InjectTrusted injects further trusted assertions for OpenSysDatabase. Returns a restore function to reinstate the previous set.
```

Comment:
Would be good to mention that this should probably be called at once once, otherwise clients would have to be *very* careful about the order things are restored.